projects/ontimize-web-ngx/src/lib/components/table/o-table.component.ts

```diff
@@ -1555,42 +1558,46 @@ export class OTableComponent extends OServiceComponent implements OnInit, OnDest
     this.queryData(void 0, queryArgs);
   }
 
-  handleClick(item: any, rowIndex: number, $event: MouseEvent) {
-    this.clickTimer = setTimeout(() => {
-      if (!this.clickPrevent) {
-        this.doHandleClick(item, rowIndex, $event);
-      }
-      this.clickPrevent = false;
-    }, this.clickDelay);
```

Comment:
This code shouldn't be removed. It prevents single click functionality to be triggered when the user performs double click